web/concrete/src/Asset/JavascriptAsset.php

```diff
@@ -70,7 +70,15 @@ protected static function process($assets, $processFunction)
             if (!file_exists($cacheFile)) {
                 $js = '';
                 foreach($assets as $asset) {
-                    $js .= file_get_contents($asset->getAssetPath()) . "\n\n";
+                    $contents = false;
+                    if ($asset->assetIsLocaleDependent()) {
+                        //@todo
```

Comment:
@aembler Which is the best way to "execute the default route function and returning the content" ?